ui/build-script.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs-extra');
+const concat = require('concat');
+
+(async function build() {
+  const files = [
+    './dist/app/runtime.js',
+    './dist/app/polyfills.js',
+    './dist/app/main.js',
+    './dist/app/scripts.js' // Include or no?
```

Comment:
I was wondering the same for vendor.js.. Seems like we need all of it. I haven't gotten it to work yet, but I'm experimenting with only using runtime and polyfills from the `'./dist/app'` directory, and then adding the bundled `'./dist/lib/atlasmap.js'` instead of having to touch and re-stitch these files back together by hand. If we can take advantage of the existing build:lib target then we'll be able to expose atlasmap as both an angular module as well as a web component, which would be awesome!